packages/discovery-provider/ddl/functions/handle_challenge_disbursements.sql

```diff
@@ -3,42 +3,6 @@ declare
   reward_manager_tx reward_manager_txs%ROWTYPE;
 	existing_notification integer;
 begin
-
```

Comment:
can we remove this trigger/file entirely?

Reply:
removing this entirely wouldn't delete the existing function on the db... i think i'll just leave this as/is cuz it's still necessary in cooldown cases.